endoreg_db/services/video_import.py

```
@@ -432,96 +818,51 @@ def _ensure_default_patient_data(self, video_instance: "VideoFile" = None) -> No
+                    # Mark sensitive meta as processed after updating missing fields
+                    state = video_file.get_or_create_state()
+                    state.mark_sensitive_meta_processed(save=True)
+                    
                     self.logger.info(f"Updated missing SensitiveMeta fields for video {video_file.uuid}: {list(update_data.keys())}")
                 except Exception as e:
                     self.logger.error(f"Failed to update SensitiveMeta for video {video_file.uuid}: {e}")
-                    
 
-    def _move_processed_files_to_storage(self, original_file_path: Union[Path, str] = None, metadata: dict = None):
+
+    def _ensure_frame_cleaning_available(self):
```

Pylint (4.0.0)

[error] method already defined line 686

(E0102)